diff --git a/postgres-appliance/Dockerfile b/postgres-appliance/Dockerfile
index 7d15482..5f2e891 100644
--- a/postgres-appliance/Dockerfile
+++ b/postgres-appliance/Dockerfile
@@ -52,7 +52,7 @@ ARG WITH_PERL=false
 ARG DEB_PG_SUPPORTED_VERSIONS="$PGOLDVERSIONS $PGVERSION"
 
 # Install PostgreSQL, extensions and contribs
-ENV POSTGIS_VERSION=3.5 \
+ENV POSTGIS_VERSION=3.6 \
     BG_MON_COMMIT=7f5887218790b263fe3f42f85f4ddc9c8400b154 \
     PG_AUTH_MON_COMMIT=fe099eef7662cbc85b0b79191f47f52f1e96b779 \
     PG_MON_COMMIT=ead1de70794ed62ca1e34d4022f6165ff36e9a91 \
